test/Elastic.Apm.Tests/MetricsTests.cs

```diff
@@ -298,22 +298,25 @@ public void CollectGcMetrics()
 				var hasGenSize = false;
 				var hasGcTime = false;
 
-
-				for (var j = 0; j < 1000; j++)
+				for (var j = 0; j < 10; j++)
 				{
-					for (var i = 0; i < 300_000; i++)
+					for (var i = 0; i < 500; i++)
 					{
-						var _ = new int[100];
+						var _ = new int[10000000];
```

Comment:
With tests running in release mode, is there a potential for the compiler to perform dead code elimination here? Perhaps  some action should be taken with the initialized array e.g. `Console.Write(...)`, to prevent elimination.

Reply:
Yeah, we can do that. I added console logging with the array.